ehrql/query_engines/base_sql.py

```diff
@@ -256,9 +256,12 @@ def get_sql_floordivide(self, node):
         float_result = self.get_sql_truedivide(node)
         return sqlalchemy.cast(SQLFunction("FLOOR", float_result), sqlalchemy.Integer)
 
+    def cast_to_int(self, value):
+        return sqlalchemy.cast(value, sqlalchemy.Integer)
+
```

Comment:
We're not consistent with this across various methods, and I dithered about it.  Mostly I went with the one that minimised duplicated code (e.g. two out of three engines used the same implementation, I left it on the base engine - partly that was because I couldn't decide how/if we should share methods between engines)

Reply:
That's a great strategy. Let's stick with it.